linker/linker_soinfo.cpp

```diff
@@ -40,6 +40,7 @@
 #include "linker_globals.h"
 #include "linker_logger.h"
 #include "linker_utils.h"
+#include "private/bionic_page.h"
```

Comment:
Just to make sure we pick up `PAGE_START()`. It used to be necessary, but now seems to be included by another header. Still think we should keep it because we directly use it.